pkg/ingester/ingester_test.go

```diff
@@ -2863,49 +2904,39 @@ func TestIngester_QueryStreamManySamples(t *testing.T) {
 	require.Equal(t, 10000+50000+samplesCount, totalSamples)
 }
 
-func TestIngester_QueryStreamManySamplesChunks(t *testing.T) {
-	// Create ingester.
-	cfg := defaultIngesterTestConfig(t)
-	cfg.StreamChunksWhenUsingBlocks = true
+func setupQueryingManySamplesAsChunksTest(ctx context.Context, t *testing.T, cfg Config) client.HealthAndIngesterClient {
+	const sampleCount = 1000 * 1000
```

Comment:
Go has another trick: `1_000_000` (just fyi, don't need to change here right now)

Reply:
Ah, I always forget about that, thanks :)